### PR TITLE
#163944862 User should be able to view/read a single article

### PR DESCRIPTION
**Pull request by malfahad**
#### What does this PR do?
Introduces functionality to let users read articles.

#### Description of Task to be completed?
- Add relevant react components for viewing articles.
- Fetch article data and display it in the views.
- Write relevant tests.

#### How should this be manually tested?
- Fetch and checkout into this branch with `git fetch origin ft-read-article-163944862 && git checkout ft-read-article-163944862`
- install dependencies with `yarn install`
- then `yarn start`
- Navigate to the `/article/article-slug` endpoint to view a single article

#### What are the relevant pivotal tracker stories?
[#163944862](https://www.pivotaltracker.com/story/show/163944862)

### CODE REVIEW

**Review by mariamiah — COMMENTED** [review]
Tested it and it works fine

**Review by armstrongsouljah — COMMENTED** [review]
I was able to view a single article and good work on getting the social icons to work on the front-end.

**Review by MuhweziDeo — DISMISSED** [review]
looks good to me

File: src/index.js [hunk 1]
```diff
@@ -7,8 +7,9 @@ import './assets/css/style.css';
 import './assets/style.scss';
 import '../node_modules/argon-design-system-free/assets/css/argon.css';
 import './assets/css/socialAuth.css';
-import 'react-toastify/dist/ReactToastify.css';
+// import '../node_modules/argon-design-system-free/assets/vendor/nucleo/css/nucleo.css'
```
Comment (RrNn) [inline src/index.js]: Remove commented out code

Comment (malfahad) [reply src/index.js]: Doing that @RrNn

**Review by MuhweziDeo — APPROVED** [review]
Works perfectly Wow @malfahad

**Review by armstrongsouljah — APPROVED** [review]
LGTM. Good work @malfahad.

**Review by calvinpete — APPROVED** [review]
LGTM

**Pull request merged**